README_for_direct_archiver.md

```diff
@@ -9,51 +19,77 @@ Follow the steps:
     - dev - https://archiver.ingest.dev.archive.data.humancellatlas.org
     - staging - https://archiver.ingest.staging.archive.data.humancellatlas.org
     - prod - https://archiver.ingest.archive.data.humancellatlas.org
-    
-2. Get the Archiver API key (`archiver_api_key`).
+
+2. Set an environment variable for the URL of ingest archiver: `export INGEST_ARCHIVER_URL=<select one from the above list>`
+3. Set an environment variable for the used deployment environment: `export ENVIRONMENT='<env>'`
+Replace `<env>` with `dev`, `staging` or `prod` in the command above.
+
+4. Get the Archiver API key (`ARCHIVER_API_KEY`).
 ```
-aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/archiver/wrangler/secrets --query SecretString --output text | jq -jr .{env}_archiver_api_key
+export ARCHIVER_API_KEY=`aws --region us-east-1 secretsmanager get-secret-value --secret-id ingest/archiver/wrangler/secrets --query SecretString --output text | jq -jr .$ENVIRONMENT_archiver_api_key`
 ```
-Replace `{env}` with `dev`, `staging` or `prod` in the command above.
+5. Set an environment variable for the UUID of the submission for which sequence data files are to be archived: `export SUBMISSION_UUID=<replace this with the submission UUID>`
 
-3. Trigger the data archive request
+6. Trigger the data archive request
 ```
-curl -X POST {ingest_archiver_url}/archiveSubmissions/data -H 'Content-Type: application/json' -H "Api-Key:{archiver_api_key}" -d '{"sub_uuid": "{sub_uuid}"}'
+curl -X POST $INGEST_ARCHIVER_URL/archiveSubmissions/data -H 'Content-Type: application/json' -H "Api-Key:$ARCHIVER_API_KEY" -d '{"sub_uuid": "$SUBMISSION_UUID"}'
```

Comment:
Are you sure `$SUBMISSION_UUID` would be replaced with the value of the variable? It is enclosed in quotes, so it will stay as it is.

Reply:
Fixed